text/0265-event-vision.md

```diff
@@ -0,0 +1,72 @@
+# Event basics
+
+## Motivation
+
+The introduction of Events has been contentious, so we want to write down and agree on a few basics.
+
+### What are events?
+
+Events are a type of log which has a required event name and a specific structure which is implied by that event name.
+
+The event name and its implied structure make events much more suitable for observability compared to traditional logs.
```

Comment:
```suggestion
The event name and its implied structure make events much more suitable for observability compared to unstructured logs.
```

I am not sure if "traditional logs" = "unstructured logs" ?

Reply:
I renamed "traditional logs" to "generic logs"

they may also be structured logs which lack an "event name"